app/views/food_trucks/new.html.erb

```diff
@@ -1,19 +1,32 @@
+<div class="row">
 <h1>Add a New Food Truck</h1>
-
+</div>
  <%= form_for @food_truck do |f| %>
-  <%= f.label :name %>
-  <%= f.text_field :name %>
-  <br>
-  <%= f.label :avg_rating %>
-  <%= f.text_field :avg_rating %>
+  <div class= "row">
+    <div class= "field columns small-6">
+      <%= f.label :name %>
+      <%= f.text_field :name %>
+    </div>
+  </div>
   <br>
+  <div class= "row">
+    <div class= "field columns small-8">
   <%= f.label :description %>
   <%= f.text_area :description %>
+    </div>
+  </div>
   <br>
+  <div class= "row">
+    <div class= "field columns small-6">
   <%= f.label :location %>
   <%= f.text_field :location %>
+    </div>
+  </div>
   <br>
```

Comment:
I'd love to see the `<br>`'s cleared out of here and some styling done to add spacing between fields.